apps/wpcom-block-editor/src/default/features/press-this.js

```diff
@@ -3,30 +3,15 @@ import { dispatch } from '@wordpress/data';
 import { getQueryArgs } from '@wordpress/url';
 import { isEditorReady } from '../../utils';
 
-const { url, title, text, embed } = getQueryArgs( window.location.href );
+const { url } = getQueryArgs( window.location.href );
 
 if ( url ) {
 	( async () => {
 		// Wait for the editor to be initialized and the core blocks registered.
 		await isEditorReady();
 
-		const link = `<a href="${ url }">${ title }</a>`;
-
-		const blocks = [];
-
-		if ( embed ) {
-			blocks.push( createBlock( 'core/embed', { url: embed } ) );
-		}
-
-		if ( text ) {
-			blocks.push(
-				createBlock( 'core/quote', {
-					value: `<p>${ text }</p>`,
-					citation: link,
-				} )
-			);
-		}
-
-		dispatch( 'core/editor' ).resetEditorBlocks( blocks );
```

Comment:
for the few minutes between the calypso change being deployed and the wpcom-block-editor change being deployed, I think embed's won't embed anything, because it relies on `text` being set to embed the link and the calypso part will no longer pass `text` through

Reply:
great point!